### ISSUE TITLE
"No such comm" again

### ISSUE THREAD

**Issue by michaelaye** [holoviz/pyviz_comms#60]
#### ALL software version info
* holoviews 1.13.2
* notebook 6.0.3
* Python 3.7

#### Description of expected behavior and the observed behavior

Not getting errors and warnings for holoviews use in a notebook.

#### Complete, minimal, self-contained example code that reproduces the issue

This code cell is all what one currently needs to trigger errors and warnings after a restart of the notebook:
```
import holoviews as hv
hv.notebook_extension('bokeh')
```
So, procedure to trigger the error and warning in the console is

1. execute code cell
2. restart notebook
3. executing code cell again

#### Stack traceback and/or browser JavaScript console output
[I 13:48:32.046 NotebookApp] Replaying 7 buffered messages
[IPKernelApp] ERROR | No such comm target registered: hv-extension-comm
[IPKernelApp] WARNING | No such comm: hv-extension-comm

**Comment by michaelaye** [comment]
it's intriguing though that different to the older stories, now there's also ERRORs being created.

**Comment by jlstevens** [comment]
@philippjfr Does the suggestion that they are now ERRORs as opposed to warnings concern you or do you still think these messages are safe to ignore?

**Comment by philippjfr** [comment]
No, it's really just the log level that was changed it still doesn't indicate any real error condition.

**Comment by jlstevens** [comment]
In that case do you think this will ever be fixed (in which case this issue can stay open) or maybe this is an unfortunately case of `wontfix`?

**Comment by michaelaye** [comment]
aand they are back... :(

```
\[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: ad3fae9f-d861-4a71-8bf7-e466009b0650
|[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: 151ed4eb-7939-4789-aec9-552998478b1b
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: a1f8aa4a-1358-40b4-9b34-cab7ddc665d9
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: 6f7a646d-01ab-4f85-99a6-83d292c2429a
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: 42dc6bd9-8e6d-48f5-8b96-498e3b88e3d5
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: fb59eda0-c563-4360-9585-3afa6f42e1aa
```

**Comment by jbednar** [comment]
My $500 USD bounty for anyone who can make those messages go away is still open...

**Comment by jbednar** [comment]
My $500 USD bounty for anyone who can make those messages go away is still open...
